pkg/daemon/ceph/osd/init.go

```diff
@@ -67,9 +62,9 @@ func getOSDTempMonMapPath(osdDataPath string) string {
 }
 
 // create a keyring for the bootstrap-osd client, it gets a limited set of privileges
-func createOSDBootstrapKeyring(context *clusterd.Context, clusterName string) error {
+func createOSDBootstrapKeyring(context *clusterd.Context, clusterName, rootDir string) error {
 	username := "client.bootstrap-osd"
-	keyringPath := getBootstrapOSDKeyringPath(context.ConfigDir, clusterName)
+	keyringPath := path.Join(rootDir, "bootstrap-osd/ceph.keyring")
```

Comment:
Would a constant make sense for `bootstrap-osd/ceph.keyring`? (Used two times at least already)